test/test_coordinator.py

```diff
@@ -551,28 +560,18 @@ def test_send_offset_fetch_request_success(patched_coord, partitions):
         future, response) 
 
 
-@pytest.mark.parametrize('response,error,dead,reassign', [
-    #(OffsetFetchResponse[0]([('foobar', [(0, 123, b'', 30), (1, 234, b'', 30)])]),
-    # Errors.GroupAuthorizationFailedError, False, False),
-    #(OffsetFetchResponse[0]([('foobar', [(0, 123, b'', 7), (1, 234, b'', 7)])]),
-    # Errors.RequestTimedOutError, True, False),
-    #(OffsetFetchResponse[0]([('foobar', [(0, 123, b'', 27), (1, 234, b'', 27)])]),
-    # Errors.RebalanceInProgressError, False, True),
```

Comment:
What was the initial purpose of these? Why were they commented out?

Reply:
I'm not sure and so I deleted them.